dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04220RemoveDeleteInactiveClusterServersJob.java

```diff
@@ -0,0 +1,64 @@
+package com.dotmarketing.startup.runonce;
+
+import java.util.List;
+
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+
+/**
+ * This task removes the database information associated to the
+ * {@code com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob} class that was deleted as of
+ * dotCMS 4.2. This will make sure that upgrading environments will not be trying to start a Quartz
+ * Job that doesn't exist anymore.
+ * 
+ * @author Jose Castro
+ * @version 4.2
+ * @since Sep 28, 2017
+ *
+ */
+public class Task04220RemoveDeleteInactiveClusterServersJob extends AbstractJDBCStartupTask {
+
+    // Execution order of the queries is important
+    private static final List<String> QUERIES = CollectionsUtils.list(
+                    "DELETE FROM qrtz_excl_fired_triggers WHERE trigger_name = 'trigger23'",
+                    "DELETE FROM dotcms.qrtz_excl_cron_triggers WHERE trigger_name = 'trigger23'",
+                    "DELETE FROM dotcms.qrtz_excl_triggers WHERE trigger_name = 'trigger23' AND job_name = 'RemoveInactiveClusterServerJob'",
+                    "DELETE FROM dotcms.qrtz_excl_job_details WHERE job_name = 'RemoveInactiveClusterServerJob' AND job_class_name = 'com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob'");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 197 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)